frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTable.java

```diff
@@ -195,8 +205,9 @@ public void refreshModel() {
   }
 
   @UIEffect
-  private void setTextForEmptyGraph(String upperText, String lowerText) {
-    getEmptyText().setText(upperText).appendSecondaryText(lowerText, StatusText.DEFAULT_ATTRIBUTES, /* listener */ null);
+  private void setTextForEmptyGraph(String upperText, String lowerText, Runnable action) {
```

Comment:
Rename `action` so as to make it clear when this Runnable is going to be executed